rpm/libphonenumber.spec

```diff
@@ -1,14 +1,11 @@
 Name:           libphonenumber
 Summary:        A library for manipulating international phone numbers
-Version:        8.12.57
+Version:        8.13.6
 Release:        1
 License:        ASL 2.0 and BSD and MIT
 URL:            https://github.com/google/libphonenumber
 Source0:        %{name}-%{version}.tar.gz
-
-# https://github.com/google/libphonenumber/pull/2556
-Patch1:         0001-Fix-geocoding-build-when-static-libraries-are-off.patch
```

Comment:
If not needed should delete the patch file too?

Reply:
Removed